test/unit/services/github_test.js

```diff
@@ -12,6 +12,9 @@ describe('#getPullRequests', function() {
     before(() => {
         sinon.stub(axios, 'get').resolves({ data: { items: items } });
     });
+    after(() => {
+        sinon.restore();
```

Comment:
C'est pas fait par défaut?

Reply:
nope les tests cassaient à cause de cela

Reply:
Il faudrait en fait définir un `afterEach` global dans un fichier indiqué en `require` à `mocha`.

Reply:
j'ai ajouté un `test-helper` comme sur `pix` mais pas sûr que c'est ce que tu avais en tête @jonathanperret

Reply:
Ça marche aussi avec le `test-helper`, c'est juste que l'utilisation de l'option `require` de `mocha` évite d'avoir à ajouter un `require` en tête de chaque test. D'autant qu'on a déjà dans ces tests un `require` superflu pour `describe` et `it`… mais ce n'est pas fondamental 🤷